MAUI/ProgressBar/States.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: States in .NET MAUI ProgressBar control | Syncfusion
+description: Learn here all about States support in Syncfusion Xamarin ProgressBar (Progress Bar) control and more.
+platform: MAUI
+control: ProgressBar
+documentation: ug
+---
+
+# States in .NET MAUI ProgressBar (Progress Bar)
+
+You can configure the states of the progress bar control depending on the usage.
```

Comment:
Configure the state of the progress bar control depending on the usage.

Reply:
done